project.clj

```diff
@@ -5,7 +5,9 @@
             :url "http://www.iplantcollaborative.org/sites/default/files/iPLANT-LICENSE.txt"}
   :deploy-repositories [["releases" :clojars]
                         ["snapshots" :clojars]]
-  :plugins [[test2junit "1.2.2"]]
+  :plugins [[jonase/eastwood "0.3.11"]
+            [test2junit "1.2.2"]]
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [korma "0.4.3"]
-                 [org.postgresql/postgresql "9.4.1212"]])
+                 [org.postgresql/postgresql "9.4.1212"]]
+  :eastwood {:exclude-linters [:def-in-def :unlimited-use]})
```

Comment:
These changes allow me to run `lein eastwood` to find common sources of bugs in the code. There were some existing `:def-in-def` warnings, which are common in code that uses `sqlkorma`. There was also existing `:unlimited-use` warning.